modules/claims_api/config/routes.rb

```diff
@@ -51,6 +51,7 @@
         post '/:veteranId/2122a', to: 'individual#submit'
         get '/:veteranId/power-of-attorney/:id', to: 'base#status'
         post '/:veteranId/power-of-attorney-request', to: 'request#request_representative'
+        post '/power-of-attorney-requests', to: 'request#index'
```

Comment:
POST is an unfortunate byproduct of needing a request body for POA codes (and later for filtering). Considered a GET request with params in the URL, but imo the complexity risk outweighs the benefit of using the conventional HTTP method, especially if the params are nested.